src/main/java/com/pelensky/contactmanager/CLInterface.java

```diff
@@ -0,0 +1,80 @@
+package com.pelensky.contactmanager;
+
+import java.io.*;
+import java.util.Scanner;
+
+
+class CLInterface {
+
+    Scanner input;
+    PrintStream output;
+    private String selectedOption;
+    private Boolean appRunning = true;
+    private ContactList contactList;
+
+
+
+    CLInterface(Scanner input, PrintStream output, ContactList contactList) {
+        this.input = input;
+        this.output = output;
+        this.contactList = contactList;
+    }
+
+    void runApp() {
+        printToConsole("Contact Manager");
+        while (appRunning) {
+            printToConsole("Type `new` to add a new contact\nType `quit` to quit.");
+            makeSelection();
+        }
+    }
+
+
+    private void printUsersSelection() {
+        printToConsole(selectedOption);
+    }
+
+    private void makeSelection() {
+        String selection = input.nextLine().toLowerCase();
+        switch (selection) {
+            case "quit": selectedOption = "See you next time!";
+                appRunning = false;
+                printUsersSelection();
+                break;
+            case "new": selectedOption = "Add a new contact";
+                printUsersSelection();
+                addNewContact();
+                break;
+            default: selectedOption = "I didn't quite get that";
+                printUsersSelection();
+                break;
+        }
+    }
+
+    private void addNewContact() {
+        printToConsole("First Name: ");
```

Comment:
Lots of repetition in this method (repetition being any repeated blocks of statements, disregarding the data passed in).